### ISSUE TITLE
Accessibility locked for the app on S23 Ultra

### ISSUE THREAD

**Issue by clotobee** [tribalfs/GalaxyMaxHzPub#117]
Sorry for posting here probably wrong but I cannot manage to enable accessibility for the GalaxyMaxHz app on my S23 Ultra. The option is greyed and it says that I need to change something in Settings > Applications but I cannot figure out what do I have to change. Please help

**Comment by tribalfs** [comment]
Just follow what's indicated in the dialog the shows.

**Comment by clotobee** [comment]
The indications that provides are no sense. It mention a section which does not really exists.

**Comment by clotobee** [comment]
Ok thanks
